docs/content/en/docs/pipeline-stages/deployers/docker.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Docker"
+linkTitle: "Docker"
+weight: 20
+featureId: deploy.docker
+---
+
+{{< alert title="Note" >}}
+This feature is currently experimental and subject to change.
+{{< /alert >}}
+
+## Deploying applications to a local Docker daemon
+
+For simple container-based applications that don't rely on
+Kubernetes resource types, Skaffold can "deploy" these applications
+by running application containers directly in your local Docker daemon.
+This enables application developers who are not yet ready to make the jump
+to Kubernetes to take advantage of the streamlined development experience
+Skaffold provides.
+
+Additionally, deploying to Docker bypasses the overhead of pushing
+images to a remote registry, as well as the 
+
+### Configuration
```

Comment:
Is this meant to lead into "Configuration" like this? If so, I think we should just have "as well as the configuration." on the same line before the next header. Users might be confused into thinking this line was unfinished

Reply:
sorry no idea how the end of this line got cut off. I finished the sentence so it should make more sense now :)